packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -504,717 +569,660 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
+                        } else if (isCalAIAction(val.value)) {
+                          setIsPhoneNumberNeeded(false);
+                          setIsSenderIsNeeded(false);
+                          setIsEmailAddressNeeded(false);
+                          setIsEmailSubjectNeeded(false);
+                          form.setValue(`steps.${step.stepNumber - 1}.emailSubject`, null);
+                          form.setValue(`steps.${step.stepNumber - 1}.reminderBody`, null);
+                          form.setValue(`steps.${step.stepNumber - 1}.sender`, null);
+                        } else {
+                          setIsPhoneNumberNeeded(false);
+                          setIsSenderIsNeeded(false);
+                          setIsEmailAddressNeeded(val.value === WorkflowActions.EMAIL_ADDRESS);
+                          setIsEmailSubjectNeeded(true);
+                          form.setValue(`steps.${step.stepNumber - 1}.agentId`, null);
+                        }
+
+                        form.setValue(`steps.${step.stepNumber - 1}.sendTo`, null);
+                        form.clearErrors(`steps.${step.stepNumber - 1}.sendTo`);
+                        form.setValue(`steps.${step.stepNumber - 1}.action`, val.value);
+                        setUpdateTemplate(!updateTemplate);
+                      }
+                    }}
+                    defaultValue={selectedAction}
+                    options={actionOptions
+                      ?.filter((option) => {
+                        if (
+                          (isCalAIAction(option.value) && form.watch("selectAll")) ||
+                          (isCalAIAction(option.value) && props.isOrganization)
+                        ) {
+                          return false;
+                        }
+                        return true;
+                      })
+                      ?.map((option) => ({
+                        ...option,
+                        creditsTeamId: teamId ?? creditsTeamId,
+                        isOrganization: props.isOrganization,
+                      }))}
+                  />
+                );
+              }}
+            />
+          </div>
+          {isCalAIAction(form.getValues(`steps.${step.stepNumber - 1}.action`)) && !stepAgentId && (
+            <div className="bg-muted mt-2 rounded-lg p-4">
+              <div className="flex items-center justify-between">
+                <div>
+                  <h2 className="text-emphasis text-sm font-medium leading-none">
+                    {t("cal_ai_agent")}
+                    <Badge startIcon="info" className="ms-2 rounded-md" variant="warning">
+                      {t("set_up_required")}
+                    </Badge>
+                  </h2>
+                  <p className="text-muted mt-2 text-sm font-medium leading-none">
+                    {t("no_phone_number_connected")}.
+                  </p>
```

Comment:
![Image](https://github.com/user-attachments/assets/4f39234f-9108-4a51-91dc-8bc8632181d3)

Padding, border radius and button are different in design

![Image](https://github.com/user-attachments/assets/39fdf8c8-6e3c-4c03-bb22-2933230f2820)